src/styles/_header.scss

```diff
@@ -0,0 +1,62 @@
+@use 'variables';
+@use 'fonts';
+
+.header {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+
+  height: variables.$headerHeight;
+  padding: 0 50px;
+
+  &__img {
+    width: 40px;
+    height: 40px;
+    text-align: center;
+  }
+
+  &__nav {
+    display: flex;
+    justify-content: flex-end;
+    align-items: stretch;
+    height: variables.$headerHeight;
```

Comment:
Ensure that `variables.$headerHeight` is defined in the `variables` file. If it's not defined, this will cause an error during compilation.